pay_ios/ios/Classes/ApplePayButtonView.swift

```diff
@@ -173,10 +173,13 @@ extension PKPaymentButtonType {
     case "donate":
       return .donate
     case "checkout":
+        guard #available(iOS 12.0, *) else { return nil }
```

Comment:
Check the indentation.

Reply:
FIxed in ca9d1002773af4c7a39cf53dfea224761dbb9d30